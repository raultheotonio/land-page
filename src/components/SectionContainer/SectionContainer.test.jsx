import { screen } from '@testing-library/react';
import { SectionContainer } from '.';
import renderTheme from '../../utils/RenderTheme';

describe('<SectionContainer />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <SectionContainer>
        <h1>children</h1>
      </SectionContainer>
    );

    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
