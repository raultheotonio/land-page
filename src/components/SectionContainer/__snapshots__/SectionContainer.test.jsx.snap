// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionContainer /> should render 1`] = `
<div>
  <div
    class="sc-bcXHqe xjnZG"
  >
    <h1>
      children
    </h1>
  </div>
</div>
`;
